src/app/public/src/modules/nav/nav.component.ts

```diff
@@ -32,8 +32,10 @@ export class StacheNavComponent implements OnInit, StacheNav {
     return Array.isArray(route.children);
   }
 
-  public navigate(route: any): void {
-    this.navService.navigate(route);
+  public navigate(event: KeyboardEvent, route: any): void {
```

Comment:
is this still necessary if we are using the `StacheLinkDirective` ?

Reply:
I'll look into this, but you're probably right.

Reply:
Removing the `click` event from the `nav.component.html` this change may not be necessary :)

Reply:
We need the event to stop the page from navigating internally when we ctrl+click. Without the event it opens a new tab AND navigates on the current page.